lib/lastfm/http_client.rb

```diff
@@ -0,0 +1,57 @@
+module Lastfm
+  class HttpClient
+    def self.null(response_body)
+      new(StubbedConnection.new(response_body))
+    end
+
+    def initialize(connection = nil)
+      @connection = connection || build_connection
+    end
+
+    def get(entity, query_params)
+      entity.new(response_body(query_params))
+    end
+
+    private
+
+    def build_connection
+      Faraday.new(url: "http://ws.audioscrobbler.com") do |faraday|
```

Comment:
if accepted my suggestion about constants

```suggestion
      Faraday.new(url: URL) do |faraday|
```

Reply:
Love it!